src/test/resources/pages/InstructorSearchPageSearchStudentsForStudent2WithExactString.html

```diff
@@ -0,0 +1,227 @@
+          <table class="table table-bordered table-striped table-responsive margin-0">
+            <thead class="background-color-medium-gray text-color-gray font-weight-normal">
+              <tr id="resultsHeader-0">
+                <th>
+                  Photo
+                </th>
+                <th class="button-sort-none" id="button_sortsection-0" onclick="toggleSort(this)">
+                  Section
+                  <span class="icon-sort unsorted">
+                  </span>
+                </th>
+                <th class="button-sort-none" id="button_sortteam-0" onclick="toggleSort(this)">
+                  Team
+                  <span class="icon-sort unsorted">
+                  </span>
+                </th>
+                <th class="button-sort-none" id="button_sortstudentname-0" onclick="toggleSort(this)">
+                  Student Name
+                  <span class="icon-sort unsorted">
+                  </span>
+                </th>
+                <th class="button-sort-none" id="button_sortstudentstatus" onclick="toggleSort(this)">
+                  Status
+                  <span class="icon-sort unsorted">
+                  </span>
+                </th>
+                <th class="button-sort-none" id="button_sortemail-0" onclick="toggleSort(this)">
+                  Email
+                  <span class="icon-sort unsorted">
+                  </span>
+                </th>
+                <th>
+                  Action(s)
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr class="student_row" id="student-c0.0">
+                <td id="studentphoto-c0.0">
+                  <div class="profile-pic-icon-click align-center" data-link="/page/studentProfilePic?studentemail=${student.email.enc}&courseid=${course.id.enc}&user=searchUI.idOfInstructor1OfCourse1">
+                    <a class="student-profile-pic-view-link btn-link">
+                      View Photo
+                    </a>
+                    <img alt="No Image Given" class="hidden" src="">
+                  </div>
+                </td>
+                <td id="studentsection-c0.0">
+                  Section 1
+                </td>
+                <td id="studentteam-c0.0.0">
+                  Team 1.1&lt;/td&gt;&lt;/div&gt;'"
+                </td>
+                <td id="studentname-c0.0">
+                  <span class="highlight">
+                    student2
+                  </span>
+                  <span class="highlight">
+                    2
+                  </span>
+                  <span class="highlight">
+                    In
+                  </span>
+                  <span class="highlight">
+                    Course1
+                  </span>
+                </td>
+                <td class="align-center">
+                  Jo
+                  <span class="highlight">
+                    in
+                  </span>
```

Comment:
Hmm I guess this feature still doesn't really work as intended then :stuck_out_tongue: ideally it shouldn't be highlighting stuff that isn't relevant to the search results, so highlighting `in` in `Joined` is not a good thing.

See if there's a way to highlight just the exact matches.